lambda/templates/repo-config/package.json.hbs

```diff
@@ -1,23 +1,38 @@
 {
   "name": "{{ kebabCase criName }}",
   "description": "",
+  "author": "GOV.UK One Login",
+  "workspaces": [
+    "lambdas/*"
+  ],
   "scripts": {
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
     "unit": "jest --silent",
     "test": "npm run unit",
     "test:coverage": "npm run unit -- --coverage",
     "pre-commit": "pre-commit run --all-files --show-diff-on-failure",
-    "pc": "npm run pre-commit"
+    "pc": "npm run pre-commit",
+    "sam:validate": "cd infrastructure && sam validate && sam validate --lint",
+    "sam:build": "cd infrastructure && sam build"
+  },
+  "dependencies": {
+    "@aws-lambda-powertools/commons": "^1.14.2",
```

Comment:
We're using dependabot, so we should install without the caret ranges - that ensures we're installing the expected version if we ever have to regenerate the lockfile.

Reply:
OK, I think it's useful to get minor version updates automatically but I'm not precious about it